src/main/java/edu/ie3/datamodel/io/factory/input/NodeInputFactory.java

```diff
@@ -17,10 +17,10 @@
 
 public class NodeInputFactory extends AssetInputEntityFactory<NodeInput, AssetInputEntityData> {
   private static final String V_TARGET = "vtarget";
-  public static final String V_RATED = "vrated";
+  public static final String V_RATED = "vRated";
   private static final String SLACK = "slack";
   private static final String GEO_POSITION = "geoposition";
-  public static final String VOLT_LVL = "voltlvl";
+  public static final String VOLT_LVL = "voltLvl";
```

Comment:
Why this change? This someone destroys the initial concept that everything should be considered lowercase in the factories as well as introduces a confusing `toLowerCase()` below, which in fact could be avoided by just writing it lowercase. Especially because the only other place it is used ist the `EntityProcessor` class where all calls are together with an `equalsIgnoreCase()`. Hence I actually see no sense in changing this here as well. Please let me know why this is necessary.

Reply:
I also feel, that it is not too nice. Basically we can break it down, that the factories and processors have to have knowledge about the field names of an entity. This information is unfortunately taken from different sources - the factories utilize the static final fields in the factories themselves, whereas the processors reflect the object's bean properties. As well unfortunately, they both cover inconsistent information (e.g. `geoposition` vs. `geoPosition`).

At this particular place it makes a difference, because the static fields `VOLT_LVL` and `V_RATED` are utilized directly to decompose the `VoltageLevel` object into two fields (cf. the `public` descriptor). Therefore, the headline of a `NodeInput` would be something like `...,slack,geoPosition,voltlvl,vrated,...` instead of `...,slack,geoPosition,voltLvl,vRated,...`.

So, all in all to me this was less worse way of fixing the headline while maintaining the correct function of the factory. But I'm open to other suggestions.

Reply:
I added #91 to keep an eye on it. 👀